### ISSUE TITLE
take image masks into account

### ISSUE THREAD

**Issue by ameisner** [jeff-regier/KillerAsteroids.jl#24]
Each WISE exposure comes with a corresponding bitmask image flagging bad pixels. I think it makes sense to keep track of these bad pixels and simply leave them out of the likelihood computation altogether. We could handle this by having type Image also store a mask. I can try to add this.

**Comment by jeff-regier** [comment]
Yep, makes sense. I think just add `mask::Bool` to Image (or is a mask not just binary?) and then skip the "masked" pixels in the loop around line 54 of `model_probability.jl`.

**Comment by ameisner** [comment]
Well, the published mask file has many different bits, e.g. 2^0 for cosmic rays, 2^1 for static bad pixels, 2^2 for saturated pixels, etc...When I write the utilities to load in the mask files, I may want to enforce some predefined definition of "good" versus "bad" pixels based on a combination of all bits, and then downstream analyses would only see binary mask images. But I guess that's not set in stone until I put the basic WISE utilities in place...
